app/lib/feed_manager.rb

```diff
@@ -165,7 +165,9 @@ def filter_from_home?(status, receiver_id)
       should_filter &&= status.account_id != status.in_reply_to_account_id                                               # and it's not a self-reply
       return should_filter
     elsif status.reblog?                                                                                                 # Filter out a reblog
-      should_filter   = Block.where(account_id: status.reblog.account_id, target_account_id: receiver_id).exists?        # or if the author of the reblogged status is blocking me
+      src_id = status.account_id
```

Comment:
Why is this a variable? Seems like it's only used in one place.

Reply:
Mostly because if I don't do this, it pushes the comments column out even farther, which seemed undesirable, but I can do whatever you'd like with it.